414.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var thirdMax = function (nums) {
+  const sorted = [...nums].sort((a, b) => a - b);
```

Comment:
You can create an array of distinct elements by using `Set`.

i.e.) `[...new Set([1, 2, 3, 3, 4, 4])]` -> `[1, 2, 3, 4]`

Reference: [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)